bentoml/yatai/yatai_service_impl.py

```diff
@@ -148,7 +148,7 @@ def ApplyDeployment(self, request, context=None):
             return ApplyDeploymentResponse(status=e.status_proto)
         except Exception as e:
             logger.error("URPC ERROR ApplyDeployment: %s", e)
-            return ApplyDeploymentResponse(status=status_pb2.Status.INTERNAL)
+            return ApplyDeploymentResponse(status=Status.INTERNAL(str(e)))
```

Comment:
the same needs to be fixed in DeleteDeployment below, it should be `status=Status.INTERNAL()` in DeleteDeployment